lib/transover_core.js

```diff
@@ -7,22 +7,21 @@ import XRegExp from 'xregexp/src'
 const Core = {}
```

Comment:
most of the changes in this file is due to formatting. I'll annotate the meaning changes below

Reply:
💀  I was wondering why this was such a large diff :/